samples/client/petstore/java/jersey3/build.sbt

```diff
@@ -0,0 +1,30 @@
+lazy val root = (project in file(".")).
+  settings(
+    organization := "org.openapitools",
+    name := "petstore-jersey3",
+    version := "1.0.0",
+    scalaVersion := "2.11.4",
+    scalacOptions ++= Seq("-feature"),
+    Compile / javacOptions ++= Seq("-Xlint:deprecation"),
+    Compile / packageDoc / publishArtifact := false,
+    resolvers += Resolver.mavenLocal,
+    libraryDependencies ++= Seq(
+      "com.google.code.findbugs" % "jsr305" % "3.0.0",
+      "io.swagger" % "swagger-annotations" % "1.6.5",
+      "org.glassfish.jersey.core" % "jersey-client" % "2.35",
```

Comment:
SBT still referring to Jersey 2.35

Reply:
Good catch. I'll update it.

Reply:
done via https://github.com/OpenAPITools/openapi-generator/pull/12046/commits/4f8519c2de74e04a056f7abd351a5a500159f971